package.json

```diff
@@ -94,7 +94,13 @@
                     ]
                 }
             ],
-            "category": "Network"
+            "category": "Network",
+            "extendInfo": {
+                "NSMicrophoneUsageDescription": "This app needs access to the microphone",
+                "NSCameraUsageDescription": "This app needs access to the camera",
```

Comment:
would be better to give them more descriptive descriptions

Reply:
I copied them from Discord. Should I go with the previous version (before 1953e4e07807597ffa9a7db653741b087881561c)?

Reply:
discord moment 😭

Reply:
Didn't really answer my question 😛